src/components/Badge/Badge.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+
+import Badge, { BadgeType } from "./Badge";
+
+<Meta
+  title="Badge"
+  component={Badge}
+  argTypes={{
+    badgeType: {
+      control: {
+        options: BadgeType,
+        type: "select",
+      },
+      defaultValue: BadgeType.UNDEFINED_LARGE_NUMBER,
+    },
+  }}
+/>
+
+export const Template = (args) => <Badge {...args} />;
+
+### Badge
+
+This is a [React](https://reactjs.org/) component for the Vanilla [Badge](https://vanillaframework.io/docs/patterns/badge).
+
+It can be used to display a numeric values.
+
+### Props
+
+<ArgsTable of={Badge} />
+
+### Default
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      value: 9,
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Colour Coding
+
+<Canvas>
+  <Story name="Colour Coding">
+    <Badge value={9} isNegative />
+  </Story>
+</Canvas>
+
+### Undefined Large Number
+
```

Comment:
A sentence explaining what this option is would be helpful (that it clamps and adds +).

Reply:
:+1: